ppl/src/main/antlr/OpenDistroPPLParser.g4

```diff
@@ -26,32 +26,31 @@ pplStatement
     : searchCommand (PIPE commands)*
     ;
 
-
 /** commands */
 commands
     : whereCommand | fieldsCommand | renameCommand | statsCommand | dedupCommand | sortCommand | evalCommand
     ;
 
 searchCommand
-    : (SEARCH)? fromClause                                          #searchWithoutFilter
-    | (SEARCH)? fromClause logicalExpression                        #searchFromClauseLogicExpr
-    | (SEARCH)? logicalExpression fromClause                        #searchLogicExprFromClause
+    : (SEARCH)? fromClause                                          #searchFrom
```

Comment:
Just a reminder, we may want to avoid those issues regarding index name in SQL since the very beginning of this project, such as index name with dot or colon (some user reported CloudWatch generates such index).

Reply:
Yes that's a good point. Thanks for your reminder. I've added these special cases now. They are refracted in `tableSource` item in the parser file.